content/en/docs/tasks/network/validate-dual-stack.md

```diff
@@ -134,28 +137,89 @@ status:
   loadBalancer: {}
 ```
 
-Create the following Service with the `ipFamily` field set to `IPv6`.
+Create the following Service that explicitly defines `IPv6` as the first array element in `.spec.ipFamilies`. Kubernetes will assign a cluster IP for the Service from the IPv6 range configured `service-cluster-ip-range` and set the `.spec.ipFamilyPolicy` to `SingleStack`.
 
-{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+{{< codenew file="service/networking/dual-stack-ipfamilies-ipv6.yaml" >}}
 
-Validate that the Service gets a cluster IP address from the IPv6 address block. You may then validate access to the service via the IP and port.
+By viewing the YAML for the Service, observe that the Service has the `.spec.ipFamilyPolicy` field has set to `SingleStack` and the `.spec.clusterIP` set to an IPv6 address from the IPv6 range set via `--service-cluster-ip-range` flag on kube-controller-manager.
+
+```shell
+kubectl get svc my-service -o yaml
 ```
- kubectl get svc -l app=MyApp
-NAME         TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
-my-service   ClusterIP   fe80:20d::d06b   <none>        80/TCP    9s
+
+```yaml
+kubectl get svc my-service -o yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: MyApp
+  name: my-service
+spec:
+  clusterIP: fd00::5118
+  clusterIPs:
+  - fd00::5118
+  ipFamilies:
+  - IPv6
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: MyApp
+  sessionAffinity: None
+  type: ClusterIP
+status:
+  loadBalancer: {}
+```
+
+Create the following Service that explicitly defines `PreferDualStack` in `.spec.ipFamilyPolicy`. Kubernetes will assign both IPv4 and IPv6 addresses (as this cluster has dual-stack enabled) and select the `.spec.ClusterIP` from the list of `.spec.ClusterIPs` based on the address family of the first element in the `.spec.ipFamilies` array.
+
+{{< codenew file="service/networking/dual-stack-preferred-svc.yaml" >}}
+
+Validate that the Service gets cluster IPs from the IPv4 and IPv6 address blocks using `kubectl describe`. You may then validate access to the service via the IPs and ports.
+
+{{< note >}}
+The `kubectl get svc` command will only show the primary IP in the `CLUSTER-IP` field.
+
+```shell
+$  kubectl get svc -l app=MyApp
```

Comment:
nit: You can remove the "$" prompt.

Reply:
Better to separate the command from its output (two blocks)